### PR TITLE
docs(docker): fix links

### PR DESCRIPTION
**Pull request by orionpulse6**
These two external docs links seems to be broken due to the architecture change.

### CODE REVIEW

**Comment by orionpulse6** [comment]
@melekes @andynog  PLZ have a check

**Comment by orionpulse6** [comment]
GM! Is there anything I need to fix here? @andynog

File: DOCKER/README.md
Comment (cason) [inline DOCKER/README.md:1]: We probably should point to the `v1.x` docs here.

**Review by melekes — APPROVED** [review]
👍

**Pull request merged**